src/ui.js

```diff
@@ -19,29 +19,34 @@ export const makeTable = async () => {
   const companiesToDisplay = [];
   companiesToDisplay.push(COMPANIES_TABLE_HEADERS);
 
+  // A function to format the date into hours:minutes format
+  function convertDate(date) {
+    return date.substring(11, 16);
```

Comment:
Unfortunately, this is not a robust implementation. The reason - if backend date format changes, your code here will change too. This is not ideal.

The better way to do it is to use JS’s Date APIs - and in particular toLocaleString function. Please take a look at the docs for it and change the inplementation.